### ISSUE TITLE
Warning: Received `false` for a non-boolean attribute `sm`.

### ISSUE THREAD

**Issue by Klerith** [nextui-org/nextui#161]
Hi, (awesome ui components first of all)

I'm using Next-UI with Next.js and TypeScript, as soon I use the ```<Text>``` component, I received this Warning / Error.

```
Warning: Received `false` for a non-boolean attribute `sm`.
```
I triple check the documentation and everything seems to be in order, I also create a repo for you to test the error.

[https://github.com/Klerith/nextui-text-error](https://github.com/Klerith/nextui-text-error)
Just clone / yarn install / run

I'm using the <Text> component inside my custom Navbar here
https://github.com/Klerith/nextui-text-error/blob/dd42a39e1698a48436cfcac0287ab4d0047e4bd2/components/ui/Navbar.tsx#L26

**Steps to reproduce**
1. Go to 'https://github.com/Klerith/nextui-text-error'
2. Clone
3. yarn install
4. yarn dev

**Expected behaviour**
No warning or errors using the Text component

**Screenshots**
<img width="800" alt="Screen Shot 2022-01-26 at 1 32 32 PM" src="https://user-images.githubusercontent.com/3438503/151224907-aad0b7da-1b75-4812-a111-a76858179f7b.png">

**Environment / Context**
- Desktop
- Operating System: OSX Monterey
- Browser and version: Chrome latest
- Device: IMAC

**Comment by klimsemenov86** [comment]
Yes, I have the same warning in the console.

The page is super simple:

```
import { Button, Text } from '@nextui-org/react';
import type { NextPage } from 'next';

const Home: NextPage = () => {
  return (
    <>
      <Text>Лендинг</Text>
    </>
  );
};

export default Home;
```

<img width="1440" alt="localhost:3000 2022-01-27 20-36-16" src="https://user-images.githubusercontent.com/55379313/151413156-5f907e3b-99a6-4c25-903e-dbbaffa9157a.png">

Also nextjs 12 with TS.

**Comment by Klerith** [comment]
> Hey @Klerith I'm glad you're using NextUI, we will check this as soon as possible, thanks for reporting it 🙌🏻

NextUI looks sick! love it... I'm creating an online course and one of the sections is going to be with NextUI

**Comment by jrgarciadev** [comment]
> > Hey @Klerith I'm glad you're using NextUI, we will check this as soon as possible, thanks for reporting it 🙌🏻
> 
> NextUI looks sick! love it... I'm creating an online course and one of the sections is going to be with NextUI

Hey @Klerith that's awesome, looking forward to seeing NextUI in one of your courses 🔥, this issue should already be fixed in the latest version `v1.0.2-beta.3`

Cheers!
